features/upgrade.feature

```diff
@@ -35,3 +35,16 @@ Feature: Upgrading Journals from 1.x.x to 2.x.x
             10.06.2013 15:40 He said "[this] is the best time to be alive".
             """
         Then the journal should have 2 entries
+
+    @skip_win
```

Comment:
Do these fail on Windows? We don't want to add new tests that already don't pass on Windows, while we're trying to fix the Windows tests.